Website/src/rendering/src/components/SpeakerInfoLine.tsx

```diff
@@ -0,0 +1,14 @@
+import { Field, Text } from '@sitecore-jss/sitecore-jss-nextjs';
+
+export function SpeakerInfoLine(props: { title: string; field: Field<string> }): JSX.Element {
```

Comment:
Here you define the type inline. In `SocialIcon` it is declared separately. It would be nice to be consistent across components. Our more complex components all use a separate type declaration. I think this is what we should use everywhere.